polyglot/piranha/src/lib.rs

```diff
@@ -93,11 +96,30 @@ pub fn execute_piranha(
   }
 
   // flag_cleaner.relevant_files
-  flag_cleaner
+  let summaries = flag_cleaner
     .get_updated_files()
     .iter()
     .map(PiranhaOutputSummary::new)
-    .collect_vec()
+    .collect_vec();
+  log_the_summary(&summaries);
+  return summaries;
+}
+
+fn log_the_summary(summaries: &Vec<PiranhaOutputSummary>) {
```

Comment:
Why `log_the_summary` rather than `log_summaries`? Specially if it's receiving multiple summaries (whatever that means)